common/app/views/fragments/atoms/storyquestions.scala.html

```diff
@@ -0,0 +1,26 @@
+@(storyquestions: model.content.StoryQuestionsAtom, isAmp: Boolean)(implicit request: RequestHeader)
+
+@if(!isAmp) {
+    <div class="submeta user__question">
```

Comment:
How does this look with the submeta already in the page? Would you be able to add a screenshot?